lldb/test/API/lang/swift/variables/move_function/TestSwiftMoveFunction.py

```diff
@@ -283,8 +280,7 @@ def do_check_addressonly_var_arg_test(self):
         # There is some sort of bug here. We should have the value here. For now
         # leave the next line commented out and validate we are not seeing the
         # value so we can detect change in behavior.
-        #self.assertGreater(varK.unsigned, 0, "varK not initialized")
-        self.assertIsNone(varK.value, "K is live but was moved?!")
+        self.assertGreater(varK.unsigned, 0, "varK not initialized")
 
```

Comment:
This looks good.

Reply:
Have you tested this on arm64, too?

Reply:
I haven't tested this on arm64, specifically. It might be working now, I am not sure. I assumed I would have to add GISEL support for this since we are changing to that in the future.